sdk/aqueduct/aqueduct_client.py

```diff
@@ -122,13 +122,14 @@ def __init__(
             not "PYTEST_CURRENT_TEST" in os.environ
         )
 
-        # Check if "@ file" in pip freeze requirements & warn user.
-        for requirement in infer_requirements():
-            if "@ file" in requirement:
-                warnings.warn(
-                    "You have installed at least one module from your local file system. These won't be installed in the server environment."
-                )
-                break
+        # Check if "@ file" in pip freeze requirements and warn user.
+        if not "localhost" in aqueduct_address:
+            for requirement in infer_requirements():
+                if "@ file" in requirement:
+                    warnings.warn(
+                        "Your local Python environment has at least one package installed from the local file system. These won't be installed into your server's Python environment."
```

Comment:
I'd rather we just print out each requirement name.

Reply:
Server is also kinda vague here because we might be running on Airflow, k8s, etc. Can we say "These packages won't be installed when running your workflow."